modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceNavigator.macro

```diff
@@ -99,4 +106,25 @@ definition {
 			value1 = "\RETURN");
 	}
 
+	macro sortField {
+		Click(
+			key_sortFieldName = "${sortFieldName}",
+			locator1 = "CommerceNavigation#SORT_FIELD",
+			value1 = "${sortFieldName}");
+	}
+
+	macro viewFilterResultsSpecificOrder {
+		var resultOrder = "0";
+
+		for (var searchAssetTitle : list "${resultsList}") {
+			var resultOrder = ${resultOrder} + 1;
+
+			AssertTextEquals.assertPartialText(
+				key_entryName = "${searchAssetTitle}",
+				locator1 = "CommerceNavigation#ENTRY_NAME",
+				resultOrder = "${resultOrder}",
```

Comment:
What is it for resultOrder? Why we pass it to the macro?

Reply:
resultOrder is a constant that checks searchAssetTitle's text i.e. line 359 of CPCommerceProducts.testcase. It makes sure that the text being checked is in line with the list item in searchAssestTitle